packages/fxa-content-server/app/scripts/views/password_strength/password_strength_balloon.js

```diff
@@ -11,17 +11,17 @@
  * @extends {BaseView}
  */
 
-import AuthErrors from '../../lib/auth-errors';
-import BaseView from '../base';
 import Cocktail from 'cocktail';
-import OneVisibleOfTypeMixin from '../mixins/one-visible-of-type-mixin';
-import Template from '../../templates/partial/password-strength-balloon.mustache';
+import AuthErrors from '../../lib/auth-errors';
 import XSS from '../../lib/xss';
+import Template from '../../templates/partial/password-strength-balloon.mustache';
+import BaseView from '../base';
 import ExternalLinksMixin from '../mixins/external-links-mixin';
+import OneVisibleOfTypeMixin from '../mixins/one-visible-of-type-mixin';
 
-const DELAY_BEFORE_HIDE_BALLOON_EL_MS = 500;
+const DELAY_BEFORE_HIDE_BALLOON_EL_MS = 200;
```

Comment:
Changed this to go with the TW default transition speed / what we're using for most of our other animations.